src/main/java/edu/kis/powp/jobs2d/drivers/adapter/monitor/DeviceMonitorDriver.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.drivers.adapter.monitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.singleton.DeviceMonitor;
+import edu.kis.powp.observer.Publisher;
+
+public class DeviceMonitorDriver implements Job2dDriver {
+
+    private Job2dDriver driver;
+    private static final Publisher publisher = new Publisher();
+
+    @Override
+    public void setPosition(int x, int y) {
+        driver.setPosition(x, y);
+        DeviceMonitor.getInstance(x, y);
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        driver.operateTo(x, y);
+        DeviceMonitor.getInstance(x, y);
```

Comment:
"operate to" should be monitored separately, because this is another form of "usage", rather than ONLY moving the devices' head.